appveyor.yml

```diff
@@ -8,10 +8,10 @@ environment:
     - nodejs_version: '8'
       webpack_version: latest
       job_part: test
-    - nodejs_version: '6'
+    - nodejs_version: '8'
       webpack_version: latest
       job_part: test
-    - nodejs_version: '4.3'
+    - nodejs_version: '6'
```

Comment:
Don't remove `node@4`, it will be remove when we drop node@4 support, just add `node@10`

Reply:
Sure? See the last commits and the failing test suite. The dependencies require node >= 6.

Reply:
https://circleci.com/gh/webpack-contrib/mini-css-extract-plugin/29